packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -0,0 +1,164 @@
+    uint256 livenessInterval;
+    uint256 thresholdPercentage;
+    uint256 minOwners;
+    address fallbackOwner;
+}
+
+/// @title Deploy
+/// @notice Script used to deploy and configure the Safe contracts which are used to manage the Superchain,
+///         as the ProxyAdminOwner and other roles in the system.
+contract DeployOwnership is Deploy {
+    /// @notice Internal function containing the deploy logic.
+    function _run() internal override {
+        console.log("start of Ownership Deployment");
+        deployAndConfigureFoundationSafe();
+        deployAndConfigueSecurityCouncilSafe();
+        console.log("deployed Security Council Safe!");
+        console.log("Ownership contracts completed");
+    }
+
+    /// @notice Returns a SafeConfig with similar to that of the Foundation Safe on Mainnet.
+    function _getExampleFoundationConfig() internal returns (SafeConfig memory safeConfig_) {
+        address[] memory exampleFoundationOwners = new address[](7);
+        for (uint256 i; i < exampleFoundationOwners.length; i++) {
+            exampleFoundationOwners[i] = makeAddr(string(abi.encode(i)));
+        }
+        safeConfig_ = SafeConfig({ threshold: 5, owners: exampleFoundationOwners });
+    }
+
+    /// @notice Returns a SafeConfig with similar to that of the Security Council Safe on Mainnet.
+    function _getExampleCouncilConfig() internal returns (SecurityCouncilConfig memory councilConfig_) {
+        address[] memory exampleCouncilOwners = new address[](13);
+        for (uint256 i; i < exampleCouncilOwners.length; i++) {
+            exampleCouncilOwners[i] = makeAddr(string(abi.encode(i)));
+        }
+        SafeConfig memory safeConfig = SafeConfig({ threshold: 10, owners: exampleCouncilOwners });
+        councilConfig_ = SecurityCouncilConfig({
+            safeConfig: safeConfig,
+            livenessInterval: 24 weeks,
+            thresholdPercentage: 75,
+            minOwners: 8,
+            fallbackOwner: mustGetAddress("FoundationSafe")
+        });
+    }
+
+    /// @notice Deploys a Safe with a configuration similar to that of the Foundation Safe on Mainnet.
+    function deployAndConfigureFoundationSafe() public returns (address addr_) {
+        address safe = deploySafe("FoundationSafe");
+        vm.startBroadcast();
+        SafeConfig memory exampleFoundationConfig = _getExampleFoundationConfig();
+        for (uint256 i; i < exampleFoundationConfig.owners.length; i++) {
+            _callViaSafe({
+                _safe: Safe(payable(safe)),
+                _target: safe,
+                _data: abi.encodeCall(OwnerManager.addOwnerWithThreshold, (exampleFoundationConfig.owners[i], 1))
+            });
+        }
+        _callViaSafe({
+            _safe: Safe(payable(safe)),
+            _target: safe,
+            _data: abi.encodeCall(
+                OwnerManager.removeOwner, (exampleFoundationConfig.owners[0], msg.sender, exampleFoundationConfig.threshold)
+            )
+        });
```

Comment:
Why do we do it this way instead of configuring the owners when we deploy the Safe

Reply:
Interesting point. I think I was just avoiding too many changes to `deploySafe()`, are you thinking it should accept a list of owners to add?

Reply:
Yea, `function deploySafe(string memory _name, address[] memory owners, uint256 threshold)` could be a good overload, and then the existing `deploySafe` method calls that one

Reply:
This has been done, so resolving